src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ChangeLevelCalculator.kt

```diff
@@ -12,48 +12,40 @@ class ChangeLevelCalculator(
 ) {
 
   fun calculateChangeLevel(crn: String): TierLevel<ChangeLevel> {
-    if (shouldCalculateChangeLevel(crn)) {
-
-      val points = getOasysNeedsPoints(crn) + getOgrsPoints(crn) + getIomNominal(crn)
-      val tier = calculateTier(points)
-
-      return TierLevel(tier, points)
-    }
-    return TierLevel(ChangeLevel.ZERO, 0)
-  }
-
-  private fun getIomNominal(crn: String): Int {
-    // We don't care about the full list, only if there is IOM Nominal
-    return if (communityApiDataService.getComplexityFactors(crn).any { it == ComplexityFactor.IOM_NOMINAL }) 2 else 0
+    return if (hasMandateForChange(crn)) {
+      val totalPoints = getOasysNeedsPoints(crn) + getOgrsPoints(crn) + getIomNominalPoints(crn)
+      val tier = when {
```

Comment:
now all the places we decide the tier level are in one place.
we'll also be adding an extra on here for the out of date assesment check